openshift/repositories-configMap.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
```

Comment:
Didn't we say all templates in thoth-application? so we maintain one place only? @goern

Reply:
ja!